src/schema/metadata/IntendedFor.yaml

```diff
@@ -0,0 +1,14 @@
+---
+key_name: IntendedFor
+description: |
+  Contains one or more filenames with paths relative to the participant subfolder.
+  The path needs to use forward slashes instead of backward slashes.
+  This field is OPTIONAL, and in case the fieldmaps do not correspond to any
+  particular scans, it does not have to be filled.
+anyOf:
+  - type: string
+    minLength: 1
```

Comment:
What if we added a `format: relative_path` option?

Reply:
I've added `dataset_relative`, `participant_relative`, and `unit` formats. They may not be used yet, but they could ultimately be useful, I think.